prac_05/birthdays.py

```diff
@@ -0,0 +1,29 @@
+"""Program that takes names and birthdates as input and returns current age"""
+
+NUM_PEOPLE = 5
+
+def main():
+    ages = {}
```

Comment:
This doesn't sound like a dictionary. If I had a list of ages, I'd call it ages.